content/en/users/tutorials/data-transfer-object-storage/_index.md

```diff
@@ -8,4 +8,143 @@ description: >
   Use EGI Data transfer to handle data in object storage
 ---
 
-TBD
+## Overview
+
+This tutorial describes the EGI Data Transfer using FTS transfers services and
+WebFTS. In the following paragraps you will learn how to:
+
+- use the [FTS](##Using-the-WebFTS-Data-Transfer-interface) command line client
+- use the [WebFTS](##FTS-client-usage) web interface
+
+to perform data transfers between a Grid storage and object storage or between
+two object storage.
+
+{{% alert title="Warning" color="warning" %}} This procedure has been tested
+with the FTS client 3.11. Older version do not support all the options
+necessary. To install the latest version please add the
+[FTS3 Production](https://fts-repo.web.cern.ch/fts-repo/fts3-prod-el7.repo)
+repository to your configuration and update the client{{% /alert %}}
+
+## Prerequisites
+
+As first step please make sure that you have installed the FTS client as
+described in [Data Transfer](../../data-transfer/), and in particular
+[Clients](../../data-transfer/clients/) for the command line FTS and to have
+your certificate installed in your browser to use
+[WebFTS](../../data-transfer/webfts/) browser based client.
+
+To access services and resources in the
+[EGI Federated Cloud](../../getting-started), you will need:
+
+- An [EGI Check-in](../../check-in) account, you can
+  [sign up here](../../check-in/signup)
+- Enrollment into a [Virtual Organisation](../../check-in/vos) (VO) that has
+  access to the services and resources you need
+- An [Object Storage](../../online-storage/object-storage/) for which you need
+  to have all the credentials available (any S3 compatible storage should work)
+- Permission to add the Object Storage credential to the FTS server or
+  alternatively for this operation you may contact support support <at> egi.eu.
```

Comment:
```suggestion
  alternatively for this operation you may contact support support **at** egi.eu.
```
Otherwise the `<at>` is understood as an HTML element. 
You may also just link to the support page https://docs.egi.eu/support/